faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -169,6 +169,14 @@ package object query {
   def Obj(pairs: (String, Expr)*): Expr =
     Expr(ObjectV("object" -> ObjectV(unwrapPairs(pairs): _*)))
 
+  /**
+    * A Null value.
+    *
+    * '''Reference''': [[https://fauna.com/documentation/queries#values]]
+    */
+  def Null(): Expr =
```

Comment:
I included that function because I realized that we didn't have a way to create nulls without importing `NullV`